moveit_ros/moveit_servo/include/moveit_servo/servo_calcs.h

```diff
@@ -106,8 +105,14 @@ class ServoCalcs
   void changeRobotLinkCommandFrame(const std::string& new_command_frame);
 
 private:
-  /** \brief Timer method */
-  void run(const ros::TimerEvent& timer_event);
+  /** \brief thread main */
```

Comment:
Let's improve these comments and rename the functions:

  /** \brief Run the main calculation loop */
  void mainCalcLoop();

/** \brief Do calculations for a single iteration. Publishes one outgoing command */
void calculateSingleIteration();